package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cocwebsite",
+  "version": "0.0.1",
+  "private": true,
+  "description": "monorepo for Community Of Coders, VJTI's website",
+  "scripts": {
+    "start:client": "yarn workspace coc-client start",
+    "start:server": "yarn workspace coc-server start:dev",
+    "dev": "concurrently --kill-others-on-fail \"yarn start:server\"  \"yarn start:client\""
+  },
+  "workspaces": [
+    "server",
+    "new_client"
```

Comment:
should be coc-client and coc-server?

Reply:
Workspaces are the folders, but we refer to a workspace by the name it has in package.json